metaflow/plugins/debug_monitor.py

```diff
@@ -1,43 +1,35 @@
-from __future__ import print_function
-
 import sys
 
-from metaflow.sidecar_messages import MessageTypes, Message
-from metaflow.monitor import Timer, deserialize_metric
-from metaflow.monitor import MEASURE_TYPE, get_monitor_msg_type
+from metaflow.sidecar import MessageTypes, Message
+from metaflow.monitor import BaseMonitor, Metric
 
 
-class DebugMonitor(object):
+class DebugMonitor(BaseMonitor):
     TYPE = "debugMonitor"
 
-    def __init__(self):
-        self.logger("init")
-
-    def count(self, count):
-        pass
+    @classmethod
+    def get_worker(cls):
+        return DebugMonitorSidecar
 
-    def measure(self, timer):
-        # type: (Timer) -> None
-        self.logger(
-            "elapsed time for {}: {}".format(timer.name, str(timer.get_duration()))
-        )
 
-    def gauge(self, gauge):
+class DebugMonitorSidecar(object):
+    def __init__(self):
         pass
 
     def process_message(self, msg):
         # type: (Message) -> None
-        self.logger("processing message %s" % str(msg.msg_type))
-        msg_type = get_monitor_msg_type(msg)
-        if msg_type == MEASURE_TYPE:
-            timer = deserialize_metric(msg.payload.get("timer"))
-            self.measure(timer)
-        else:
-            pass
-
-    def shutdown(self):
-        sys.stderr.flush()
-
-    def logger(self, msg):
-        print("local_monitor: %s" % msg, file=sys.stderr)
+        if msg.msg_type == MessageTypes.MUST_SEND:
+            self._context = msg.payload
```

Comment:
We can just print both the `MUST_SEND` and `BEST_EFFORT` messages here in this implementation

Reply:
fixed.